pym/portage/sync/modules/git/git.py

```diff
@@ -50,6 +50,11 @@ def new(self, **kwargs):
 			sync_uri = sync_uri[6:]
 
 		git_cmd_opts = ""
+		if self.repo.module_specific_options.get('sync-git-clone-env'):
+			shlexed_env = shlex_split(self.repo.module_specific_options['sync-git-clone-env'])
+			clone_env = {k: v for [k, v] in [ assignment.split('=', 1) for assignment in shlexed_env]}
```

Comment:
This will raise ValueError if any of the tokens is missing an `=` character. Here is a way to silently ignore errors:
```
dict((k, v) for k, __, v in (assignment.partition('=') for assignment in shlexed_env) if k)
```